collect_app/src/main/res/layout/drive_layout.xml

```diff
@@ -16,94 +15,96 @@ the License.
 
 
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                android:layout_width="fill_parent"
-                android:layout_height="fill_parent"
-                android:layout_alignParentTop="true" >
+    android:layout_width="fill_parent"
```

Comment:
Yes, that does make code review much easier and lowers the chance of errors. I'll think about adding a suggestion like that to the README.